framework/include/executioners/PicardSolve.h

```diff
@@ -9,7 +9,8 @@
 
 #pragma once
 
-#include "SolveObject.h"
+#include "FixedPointSolve.h"
+#include "NonlinearSystem.h"
```

Comment:
We should use `NonlinearSystemBase.h`. I would assume Picard should work for `nonlineareigensystem` as well

Reply:
Ok making that change. Though there are definitely better methods than secant and steffensen to solve eigenvalue problems.

Reply:
Thanks. Yes, just for consistency ,,